javascript/array-insert-shift/README.md

```diff
@@ -0,0 +1,8 @@
+# Insert value into an Array
+Given an array and insert a value into the array.
+
+## Whiteboard Process
+![array-insert-shift whiteboard](./array-insert-shift.png)
+
```

Comment:
If the image is to big you can do something like this 
```
<p align="center">
<img src="example.png"  width="500" >
</p> 
```

Which will center also center the image